includes/wikia/models/DesignSystemGlobalFooterModel.class.php

```diff
@@ -381,7 +381,9 @@ public function getData() {
 					'type' => 'translatable-text',
 					'key' => 'global-footer-licensing-description',
```

Comment:
If the vertical is available, this message key should be `global-footer-licensing-and-vertical-description`. Though keep in mind, about 30,000 wikis don't have a vertical, so those should still use `global-footer-licensing-description`. We previously had logic for choosing which message to use based on whether the vertical was available, which was removed in https://github.com/Wikia/app/pull/10871 when we were temporarily removing the vertical while a decision was made.